.github/workflows/tests.yml

```diff
@@ -162,4 +202,9 @@ jobs:
       - name: Select Xcode
         run: .github/workflows/xcode_select.sh
       - name: Build App
-        run: bazelisk build -s tests/ios/app/App --apple_platform_type=ios --ios_minimum_os=10.2  --ios_multi_cpus=i386,x86_64
```

Comment:
Was `-s` intentionally removed? What did it do?

Reply:
That prints the subcommands, i meant to add this `config:ci`. Do we think thats useful, it could be very verbose but its for CI so?

Reply:
Added this to `config=ci` as well as `--verbose_failures` figure its better to be verbose in CI.